controllers/chat/sendSignedMessage.js

```diff
@@ -55,7 +55,7 @@ const sendSignedMesage = async (req, res) => {
       error: validated
     });
 
-  const {channelId, message, channelType, attachments, messageType, replyMessageId} = req.body;
+  let {channelId, message, channelType, attachments, messageType, replyMessageId} = req.body;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `const` to `let` for destructuring `req.body` is unnecessary if the variables are not reassigned later in the code. If they are, please ensure that this reassignment is necessary and safe, as it can lead to unexpected behavior.
```diff
- let {channelId, message, channelType, attachments, messageType, replyMessageId} = req.body;
+ const {channelId, message, channelType, attachments, messageType, replyMessageId} = req.body;
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->